pkgs/tools/misc/contacts/default.nix

```diff
@@ -9,15 +9,15 @@ stdenv.mkDerivation rec {
     sha256 = "0wdqc1ndgrdhqapvvgx5xihc750szv08lp91x4l6n0gh59cpxpg3";
   };
 
-  preBuild = ''
-    substituteInPlace Makefile --replace "xcodebuild" "/usr/bin/xcodebuild"
-  '';
+  buildInputs = [ xcbuild Foundation AddressBook ];
 
   installPhase = ''
     mkdir -p $out/bin
-    cp ./build/Deployment/contacts $out/bin
+    cp ./contacts-*/Build/Products/Default-*/contacts $out/bin
   '';
 
+  NIX_LDFLAGS = " -F${Foundation}/Library/Frameworks/ -F${AddressBook}/Library/Frameworks/";
```

Comment:
There's definitely something weird going on. "Foundation" should be in stdenv anyway right?